### PR TITLE
net-misc/streamlink: Bump version, update deps

### PR DESCRIPTION
**Pull request by Wesmania**
Package-Manager: Portage-2.3.40, Repoman-2.3.9

### CODE REVIEW

File: net-misc/streamlink/streamlink-0.12.1.ebuild [hunk 1]
```diff
@@ -0,0 +1,59 @@
+# Copyright 1999-2018 Gentoo Foundation
```
Comment (mgorny) [inline net-misc/streamlink/streamlink-0.12.1.ebuild]: Why are you adding the old version? Is there any reason anyone would use it instead of 0.13.0?

Comment (Wesmania) [reply net-misc/streamlink/streamlink-0.12.1.ebuild]: I didn't know the convention regarding that. I'll remove it.

Comment (mgorny) [reply net-misc/streamlink/streamlink-0.12.1.ebuild]: The idea is that if you add two versions, people will just upgrade to the newer and the older will never be really used.

Comment (Wesmania) [reply net-misc/streamlink/streamlink-0.12.1.ebuild]: Fixed.

File: net-misc/streamlink/streamlink-0.13.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+KEYWORDS="~amd64 ~x86"
+LICENSE="BSD-2 Apache-2.0"
+SLOT="0"
+IUSE="doc test"
+
+RDEPEND="dev-python/pycryptodome[${PYTHON_USEDEP}]
+	dev-python/requests[${PYTHON_USEDEP}]
+	virtual/python-futures[${PYTHON_USEDEP}]
+	virtual/python-singledispatch[${PYTHON_USEDEP}]
+	dev-python/backports-shutil_which[$(python_gen_usedep 'python2*')]
```
[the remark this answers is not in the document]
Comment (Wesmania) [reply net-misc/streamlink/streamlink-0.13.0.ebuild]: Done

File: net-misc/streamlink/streamlink-0.13.0.ebuild [hunk 3]
```diff
@@ -21,7 +21,7 @@ RDEPEND="dev-python/pycryptodome[${PYTHON_USEDEP}]
 	dev-python/requests[${PYTHON_USEDEP}]
```
[the remark this answers is not in the document]
Comment (Wesmania) [reply net-misc/streamlink/streamlink-0.13.0.ebuild]: Done.

**Comment by Wesmania** [comment]
I'm not promising anything, but I have streamlink-9999 in my @world - I'll make sure to do PRs with fixes whenever I see it break (and version bumps whenever I check for releases).

**Pull request closed without merging**